CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.6.0] - 2020-03-16
+### Changed
+- README estandarizado, con links a repositorios con ejemplos utilizando distintas versiones de woocommerce + PHP.
```

Comment:
Pudieras agregar los PR relacionados. Mira la ultima entrada.

Reply:
Para editar un archivo es suficiente con subir un nuevo commit con la edición a la rama desde la que hago el PR?

Reply:
We use english for reviews and descriptions :)
You can commit directly on the branch.

Reply:
updated changelog with PR links